frontend/src/Components/Output.jsx

```diff
@@ -0,0 +1,113 @@
+import { useState } from "react";
+import { Light as SyntaxHighlighter } from "react-syntax-highlighter";
+import { docco,github,monokai } from "react-syntax-highlighter/dist/esm/styles/hljs";
+
+const Output = () => {
+  const [option, setOption] = useState("Output");
+  const [compilation, setCompilation] = useState("Not Started");
+  const [time, setTime] = useState(0);
+  const [error, setError] = useState(false);
+  const [errorCount, setErrorCount] = useState(0);
+  const [warning, setWarning] = useState(0);
+    const [errors, setErrors] = useState("");
+
+    const [output, setOutput] = useState('Not Compiled');
+    const [files, setFiles] = useState('Still Not Compiled');
+    
+    let errorText = `Warning: Each child in a list should have a unique "key" prop.
```

Comment:
Will the same error text appear for all failures?

Reply:
This is a sample error text.
The actual error will shown when the error comes from the Backend.
Same for output, Compilation Errors, Warnings, Compilation Time etc.